clickhouse/pyproject.toml

```diff
@@ -40,7 +40,8 @@ text = "BSD-3-Clause"
 
 [project.optional-dependencies]
 deps = [
-    "clickhouse-cityhash==1.0.2.3",
+    "clickhouse-cityhash==1.0.2.3; python_version < '3.0'",
+    "clickhouse-cityhash==1.0.2.4; python_version > '3.0'",
```

Comment:
I'm not sure why we do not have the same version for py2 and py3, 1.0.2.4 seems to support py2 https://github.com/xzkostyan/clickhouse-cityhash/blob/master/setup.py#L91. Any ideas?

Reply:
Since this is not a blocker and there's time pressure, ignoring it for now :(